api/scripts/makemigrations.sh

```diff
@@ -0,0 +1,5 @@
+#!bin/bash
```

Comment:
It should probably be `#!/bin/bash`

Try executing it: `./scripts/makemigrations.sh`

Reply:
> It should probably be `#!/bin/bash`
yeah 😅